### PR TITLE
Feature/solvis leaflet

### PR DESCRIPTION
**Pull request by 0qy**

### CODE REVIEW

File: src/pages/SolutionAnalysisPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+      filteredLocationIDs.push(location.id);
+    });
+    setLocationIDs(filteredLocationIDs);
+  }, [locationSelections]);
+
+  useEffect(() => {
+    const locationListID = process.env.REACT_APP_ANALYSIS_LOC_LIST_ID as string;
+    const radiiID = process.env.REACT_APP_RADII_ID;
+
+    axios
```
Comment (wooneh) [inline src/pages/SolutionAnalysisPreview.tsx:71]: Interested in the use of `axios` here - I believe we've already used `fetch` for our purposes, is there a reason for the new library?

Comment (0qy) [reply src/pages/SolutionAnalysisPreview.tsx:80]: Do you think I should use fetch instead for these api calls and no use a new library?

**Review by wooneh — APPROVED** [review]

File: src/pages/SolutionAnalysisPreview.tsx (hunk 1 above)
Comment (wooneh) [reply src/pages/SolutionAnalysisPreview.tsx:80]: No I think it should be fine, was just curious.

**Comment by 0qy** [comment]
@wooneh If you get the chance this PR could be reviewed again, we added the preview feature into the Inversion Solutions view.

File: src/pages/SolutionAnalysisPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      </FloatingCard>
+      <MapContainer center={nz_centre} zoom={zoom} scrollWheelZoom={true} style={{ height: '700px' }}>
+        <TileLayer url={provider_url} />
+        {geojsonData && <GeoJSON key={Math.random()} data={geojsonData} style={myStyle} />}
+      </MapContainer>
+    </>
+  );
+};
+
+export default SolutionAnalysisPreview;
```
Comment (chrisbc) [inline src/pages/SolutionAnalysisPreview.tsx:206]: typo @0qy !!

File: tsconfig.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
   },
   "include": [
     "src/**/*",
-    "*.config.js"
+    "*.config.js",
+    "decs.d.ts"
```
Comment (chrisbc) [inline tsconfig.json:26]: @0qy remove?

**Review by chrisbc — APPROVED** [review]
@0qy all good with the 2 comments, thanks

**Pull request merged**